articles/iot-central/core/howto-transform-data.md

```diff
@@ -108,15 +108,15 @@ In this scenario, the IoT Edge device runs a custom module that transforms the d
 - Build the custom module.
 - Add the custom module to a container registry.
 
-The IoT Edge runtime downloads custom modules from a container registry such as an Azure container registry or Docker Hub. The [Azure Cloud Shell](../../cloud-shell/overview.md) has all the tools you need to create a container registry, build the module, and upload the module to the registry:
+The IoT Edge runtime downloads custom modules from a container registry such as an Azure Container Registry or Docker Hub. The [Azure Cloud Shell](../../cloud-shell/overview.md) has all the tools you need to create a container registry, build the module, and upload the module to the registry:
 
 To create a container registry:
 
 1. Open the [Azure Cloud Shell](https://shell.azure.com/) and sign in to your Azure subscription.
 
 1. Select the **Bash** shell.
 
-1. Run the following commands to create an Azure container registry:
+1. Run the following commands to create an Azure Container Registry:
```

Comment:
This one should be left as is. When we refer to an instance we use lower case. Uppercase is reserved for when we refer to the service. Take a look at https://learn.microsoft.com/en-us/azure/container-registry/container-registry-intro for examples.